blathering-barnacles/Bangazon-2
# Ticket10

Description: Added the ability to cancel an order.

Steps to Test:
* Navigate to shopping cart, click on cancel order, the items there in should disapear.
* Check your database, your ecomm_order should now have a deletedOn date.
* Check your ecomm_productorder table, the items with the order number should now have a deletedOn date.

Link to Feature Ticket:[Ticket10](https://github.com/blathering-barnacles/Bangazon-2/issues/10)
